### PR TITLE
Add weekly highlights 2023-05-18

### PR DESCRIPTION
**Pull request by mproffitt**
This weeks highlights PR

### CODE REVIEW

File: src/content/changes/highlights/2023-05-18.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: Highlights for the week ending May 18 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+- All management-cluster Flux installations (`flux-system`, `flux-giantswarm`) have been upgraded to Flux [v0.41.2](https://github.com/fluxcd/flux2/releases/tag/v0.41.2).
```
Comment (pipo02mix) [inline src/content/changes/highlights/2023-05-18.md:12]: From which version? any feature/bug to highlight?

File: src/content/changes/highlights/2023-05-18.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- All management-cluster External Secret installations have been upgraded to External Secrets [0.8.1](https://github.com/external-secrets/external-secrets/releases/tag/v0.8.1)
```
Comment (pipo02mix) [inline src/content/changes/highlights/2023-05-18.md:13]: Same here, we can highlight anything?

File: src/content/changes/highlights/2023-05-18.md (hunk 1 above)
Comment (mproffitt) [reply src/content/changes/highlights/2023-05-18.md:12]: Updated to include key highlights

File: src/content/changes/highlights/2023-05-18.md (hunk 2 above)
Comment (mproffitt) [reply src/content/changes/highlights/2023-05-18.md]: Updated to include key highlights

**Review by pipo02mix — APPROVED** [review]
LGTM not sure if other POs want to add something

**Comment by pipo02mix** [comment]
@mproffitt I think we are good to ship it :)

File: src/content/changes/highlights/2023-05-18.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- [flux-app](https://github.com/giantswarm/flux-app) version [v0.23.1](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0231---2023-05-04) upgrades to support Flux toolkit [v0.41.2](https://github.com/fluxcd/flux2/releases/tag/v0.41.2) introduces flux events command to display events for flux resources, improves memory consumption for Helm controller with up to 50% observed reduction and now includes an opt-in OOM watcher allowing for more graceful termination within helm controller.
+- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.5](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0145---2023-05-16) introduces a new Job for ensuring Kyverno webhooks are properly deleted when uninstalling the App and replaces a deprecated label.
+- [security-bundle](https://github.com/giantswarm/security-bundle) version [0.14.3](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#0141---2023-05-04) (and 0.14.2) includes the Kyverno release from this announcement as well as an update to the Trivy app which makes its VPA behavior configurable.
+
```
Comment (stone-z) [inline src/content/changes/highlights/2023-05-18.md:22]:
```suggestion
## Apps
- [crossplane](https://github.com/giantswarm/crossplane) version [v2.2.0](https://github.com/giantswarm/crossplane/blob/main/CHANGELOG.md#220---2023-05-09) upgrades crossplane to version [v1.11.3](https://github.com/crossplane/crossplane/releases/tag/v1.11.3) This is a scoped patch release to better support composition functions
- [external-secrets](https://github.com/giantswarm/external-secrets/) version [v0.5.0](https://github.com/giantswarm/external-secrets/blob/main/CHANGELOG.md#050---2023-04-27) upgrades External Secrets operator to version [v0.8.1](https://github.com/external-secrets/external-secrets/releases/tag/v0.8.1) introduces a new generator for Vault dynamic secrets and allows for the inclusion of metadataPolicy Fetch for supported backend providers
- [flux-app](https://github.com/giantswarm/flux-app) version [v0.23.1](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0231---2023-05-04) upgrades to support Flux toolkit [v0.41.2](https://github.com/fluxcd/flux2/releases/tag/v0.41.2) introduces flux events command to display events for flux resources, improves memory consumption for Helm controller with up to 50% observed reduction and now includes an opt-in OOM watcher allowing for more graceful termination within helm controller.
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.5](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0145---2023-05-16) introduces a new Job for ensuring Kyverno webhooks are properly deleted when uninstalling the App and replaces a deprecated label.
- [security-bundle](https://github.com/giantswarm/security-bundle) version [0.14.3](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#0143---2023-05-17) (and 0.14.2) includes the Kyverno release from this announcement as well as an update to the Trivy app which makes its VPA behavior configurable.

```

**Pull request merged**